docs/features/repo-pages.md

```diff
@@ -37,6 +37,10 @@ There are a couple of ways to access a repository page:
 
     [https://app.opensauced.pizza/s/vercel/next.js](https://app.opensauced.pizza/s/vercel/next.js)
 
+:::tip
+If you want to keep track of a repository, use the **Add to WorkSpace** button on the Repo page to add it to your [Workspace](https://opensauced.pizza/docs/features/workspaces/).
```

Comment:
We usually use double quotes instead of bold for the buttons' names.


```suggestion
If you want to keep track of a repository, use the "Add to WorkSpace" button on the Repo page to add it to your [Workspace](https://opensauced.pizza/docs/features/workspaces/).
```